lib/api/multiObjectDelete.js

```diff
@@ -269,6 +269,8 @@ export function getObjMetadataAndDelete(authInfo, canonicalID, request,
             // return DeleteMarkerVersionId equals the versionID of the marker
             // you just generated and DeleteMarker tag equals true
             if (deleteInfo.newDeleteMarker) {
+                // Adding a delete marker should add one to numberOfObjects
+                numOfObjects--;
```

Comment:
There is also a case when, if versioning is suspended and there is an existing null version, the new delete marker overwrites the null version, resulting in a change in the total content stored in the bucket. Is it something we need to track?